### PR TITLE
Website build

### PR DESCRIPTION
**Pull request by KanoCode**
In this milestone, the following was done:
- **Interactions**

 **Home page**

When the page loads, the web app shows the list of companies that could be filtered out by some company symbol.

When the user clicks (or taps) on a company item, the application navigates to the details page.

**Details page**

- On the details page, the web app retrieves data from the API to show detailed data in the category.
When the user clicks on the "home" button (<), the user navigates to the home page.

**Testing **

- Created unit tests for my reducers
- Created integration tests for my application using the React Testing Library.
- Created snapshot for application

### CODE REVIEW

File: src/redux/store/configureStore.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-//   missions: missionsReducer,
+  profile: ProfileReducer,
+  searchedData: SearchReducer,
+  stocks: StocksReducer,
 });
 
-const store = createStore(
-  applyMiddleware(thunk, logger),
-);
+const store = createStore(reducers, applyMiddleware(thunk, logger));
```
Comment (afanimartin) [inline src/redux/store/configureStore.js]: - `createStore` is deprecated. Please use `connect` instead

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @KanoCode, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## 👏🏿 👏🏿 👏🏿 
- Nice video presentation about the project
- Attribution of designer in the README file.

## Required Changes ♻️
- Please deploy the app and add a link to the live app in the README file.
- Add `Home` and `Detail` pages. The Home page has a list of categories that could be filtered using given parameters, e.g. category name while the Detail page contains the detail of an item.
- It would be nice to have a "Back" button from the detail page to the home page. This will improve the experience of your app.
- Running the command `npm run test` throws the error: `Error: Failed to initialize watch plugin "node_modules/jest-watch-typeahead/filename.js":`. Please make sure all test cases run.
_Check more comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/store/configureStore.js (hunk 1 above)
Comment (KanoCode) [reply src/redux/store/configureStore.js]: Thank you @afanimartin  for the feedback, I could not find any function with that name associated with redux, however, I will use configureStore from the redux toolkit, it is not deprecated.

File: src/components/LineChart.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
```
Comment (hiromataba) [inline src/components/LineChart.js:1]: - [ ] [OPTIONAL] Linting is a great tool to make your code clean so please avoid disabling them

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @KanoCode     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/LineChart.js (hunk 2 above)
Comment (KanoCode) [reply src/components/LineChart.js:1]: Thank you @hiromataba

**Pull request merged**